public/opensearch.xml

```diff
@@ -0,0 +1,8 @@
+<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/" xmlns:moz="http://www.mozilla.org/2006/browser/search/">
+  <ShortName>Egghead</ShortName>
+  <Description>Search Egghead</Description>
```

Comment:
thank you @arthurdenner, I'm just gonna suggest lowercasing egghead and it'll be good to go! :)  

```suggestion
  <ShortName>egghead</ShortName>
  <Description>Search egghead</Description>
```